examples/consumer/main.go

```diff
@@ -218,6 +221,9 @@ func getEvent(w http.ResponseWriter, req *http.Request) {
 	}
 	e := string(bodyBytes)
 	if e != "" {
+		if consumerType == HW {
```

Comment:
I believe you need this  to run consumer  for HW events

Reply:
yes just for monitoring the performance

Reply:
also you need for QE to run manuall tests